### PR TITLE
PLAT-103742: Support accessibility for IncrementSlider components

### PR DESCRIPTION
**Pull request by Baekwoo-s**
Support accessibility for IncrementSlider components

Enact-DCO-1.0-Signed-off-by: Baekwoo Jung (baekwoo.jung@lge.com)

### CODE REVIEW

File: IncrementSlider/IncrementSlider.js [hunk 1]
```diff
@@ -405,18 +407,38 @@ const IncrementSliderBase = kind({
 	},
 
 	computed: {
-		className: ({orientation, styler}) => styler.append(orientation)
+		className: ({orientation, styler}) => styler.append(orientation),
+		decrementDisabled: ({disabled, min, value = min}) => disabled || value <= min,
+		incrementDisabled: ({disabled, max, min, value = min}) => disabled || value >= max,
+		decrementAriaLabel: ({'aria-valuetext': valueText, decrementAriaLabel, min, value = min}) => {
+			if (decrementAriaLabel == null) {
+				decrementAriaLabel = $L('press ok button to decrease the value');
```
Comment (seunghoh) [inline IncrementSlider/IncrementSlider.js:415]: Can we change this string to 'press button to crease the value' or do we have any better string? Agate is touch based Auto target

File: IncrementSlider/IncrementSlider.js [hunk 2]
```diff
@@ -405,18 +407,38 @@ const IncrementSliderBase = kind({
...
+			}
+
+			return `${valueText != null ? valueText : value} ${decrementAriaLabel}`;
+		},
+		incrementAriaLabel: ({'aria-valuetext': valueText, incrementAriaLabel, min, value = min}) => {
+			if (incrementAriaLabel == null) {
+				incrementAriaLabel = $L('press ok button to increase the value');
```
Comment (seunghoh) [inline IncrementSlider/IncrementSlider.js:422]: Same as above. Can we remove 'ok'?

File: IncrementSlider/IncrementSlider.js (hunk 1 above)
Comment (Baekwoo-s) [reply IncrementSlider/IncrementSlider.js:415]: I checked other strings, but couldn't find a suitable string.
I agree to exclude "OK" from the code above.

**Comment by seunghoh** [comment]
LGTM. 
@Baekwoo-s Please take care of the xliff file for Auto

**Comment by Baekwoo-s** [comment]
@seunghoh 
Yes, I will create a ticket.

**Pull request merged**